ratpack-test/src/main/java/ratpack/test/http/internal/DefaultTestHttpClient.java

```diff
@@ -256,6 +256,7 @@ private HttpClient httpClient() {
       .byteBufAllocator(TestByteBufAllocators.LEAKING_UNPOOLED_HEAP)
       .maxContentLength(Integer.MAX_VALUE)
       .poolSize(8)
+      .addressResolver(HttpClient.blockingResolver())
```

Comment:
Since this is the blocking client for tests, use the blocking resolver and avoid the warning.